components/expenses/ExpenseForm.js

```diff
@@ -67,11 +68,18 @@ const msg = defineMessages({
     id: 'ExpenseForm.StepExpenseFundingRequest',
     defaultMessage: 'Set grant details',
   },
-
   stepPayee: {
     id: 'ExpenseForm.StepPayeeInvoice',
     defaultMessage: 'Payee information',
   },
+  resetExpense: {
+    id: 'ExpenseForm.resetExpense',
+    defaultMessage: 'Reset Form',
+  },
+  confirmResetExpense: {
+    id: 'ExpenseForm.confirmResetExpense',
+    defaultMessage: 'Do you really want to reset the expense form?',
+  },
```

Comment:
As per https://github.com/opencollective/opencollective/issues/4697#issuecomment-921858385 for new strings we don't need the `id`s. 🙂 

```suggestion
  resetExpense: {
    defaultMessage: 'Reset Form',
  },
  confirmResetExpense: {
    defaultMessage: 'Do you really want to reset the expense form?',
  },
```

Reply:
That's nice.